guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -56,6 +56,11 @@ include::topics/upgrading_discovery_satellite.adoc[leveloffset=+3]
 // Configuring Subnets with a Template Capsule
 include::topics/upgrading_discovery_subnet_with_a_template_capsule.adoc[leveloffset=+3]
 
+ifdef::katello,satellite[]
```

Comment:
@evgeni 
would you mind advising which parts are not applicable for Satellite? in the procedure in general?